lib/vsc/mympirun/rm/sched.py

```diff
@@ -49,6 +49,14 @@ def what_sched(requested):
                 return sched, found_sched
         LOGGER.warn("%s scheduler was requested, but mympirun failed to find an implementation", requested)
 
+    # if PBS_NODEFILE is not present, use local scheduler
+    filename = os.environ.get('PBS_NODEFILE', None)
+    if filename is None:
```

Comment:
also, we should only be doing this if no scheduler was specified (or if the default `pbs` scheduler was used)?